core/src/main/java/hudson/FilePath.java

```diff
@@ -2371,7 +2402,7 @@ public String validateAntFileMask(final String fileMasks, final int bound) throw
      * Default bound for {@link #validateAntFileMask(String, int, boolean)}.
      * @since 1.592
      */
-    public static int VALIDATE_ANT_FILE_MASK_BOUND = SystemProperties.getInteger(FilePath.class.getName() + ".VALIDATE_ANT_FILE_MASK_BOUND", 10000);
+    public static final int VALIDATE_ANT_FILE_MASK_BOUND = SystemProperties.getInteger(FilePath.class.getName() + ".VALIDATE_ANT_FILE_MASK_BOUND", 10000);
```

Comment:
This change is unrelated && it causes merge conflict.
Note that some fields are bob-final for a reason, so that they cannot be modified in System scripts

Reply:
reverted